pages/contribute/working_with_git.md

```diff
@@ -50,32 +50,44 @@ NOTE: if you already did these steps in the past, start from the `git fetch upst
 
 The website is build on GitHub using Jekyll, a simple, static site generator based on ruby. When you have a local copy cloned onto your computer, it is possible to generate the website based on this repo. This makes it possible to preview changes live, every time you save a file from within the GitHub rdmkit repo. Follow these steps to deploy the website based on your local clone (copy) of the rdmkit repo:
 
-1. Make sure you have cloned the rdmkit repo
-    ```
+Make sure you have cloned the rdmkit repo:
+
     git clone git@github.com:USERNAME/rdmkit.git
     cd rdmkit
+
+
+To run the website locally, you can either use [Docker](https://www.docker.com/) or use Jekyll directly after installing various dependencies.
+
+### Run using Docker
+
+1. If not already installed on your machine, install Docker. From the root of the ``rdmkit`` directory, run the following command:
     ```
+    ./docker_startup.sh
+    ```
+ 
+
+### Run using Jekyll directly
 
-2. If not already present on your machine, install ruby. Note that incompatibility issues may arise with ruby 3.0.0 (released 25.12.20) or newer versions.
+1. If not already present on your machine, install ruby. Note that incompatibility issues may arise with ruby 3.0.0 (released 25.12.20) or newer versions.
 
 
-3. Install Jekyll
-If you’ve never installed or run a Jekyll site locally on your computer, follow these instructions to install Jekyll:
+1. Install Jekyll
```

Comment:
it seems like the numbering is lost here

Reply:
nothing is lost, this is the preferred way to format numbered list. (I will open a second PR unifying that)
see 
<img width="1216" alt="Screenshot 2021-06-15 at 13 31 33" src="https://user-images.githubusercontent.com/1022396/122052823-05487300-cdde-11eb-8dbd-aa4bf5620574.png">

Reply:
Could you add a reference this? Out of curiosity. Because I know your way is supported, but I didn't know it was THE way of doing thing.

Reply:
it is preferable in the sense that if you remove an item from the enumeration you do not have to change the value of the following items
See https://guides.github.com/features/mastering-markdown/

Reply:
From a visual aspect I like it more with the real numbers since it is closer to how it looks in real life.

Reply:
but it is not really maintainable. This factor should be taken into account I think.
The "real life" output is the generated html page

Reply:
@jburel thanks!